CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 8.3.7
```

Comment:
https://pub.dev/packages/file_picker/changelog doesn't list 8.3.6 yet, so that is possibly still awaiting publish

Reply:
Edit: it seems the previous change didn't get a pubspec change, so it was not yet published. Give me a moment to address that